python/lsst/rubintv/production/rubinTv.py

```diff
@@ -455,7 +455,7 @@ def callback(self, dataId):
         try:
             tempFilename = tempfile.mktemp(suffix='.png')
             uploadFilename = _dataIdToFilename(self.channel, dataId)
-            plotted = plotMountTracking(dataId, self.butler, self.client, self.fig, tempFilename, self.log)
+            plotted = calculateMountErrors(dataId, self.butler, self.client, self.fig, tempFilename, self.log)
```

Comment:
`calculateMountError` returns the list of arrays discussed above, which will always be True-like and run the following block.  Is that intended?

Reply:
Nice catch, thanks. Have changed to just test if the output file was written.